File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/11172-Relational_Operator/Main.java
/*
 * 11172 - Relational Operators
 * Take in 2 numbers and output the relation that is appropriate
 * for the given numbers.
 * Java version 6/8/2016
 */

import java.io.PrintStream;
import java.util.Scanner;

public class Main {
    static final Scanner input = new Scanner(System.in);
    static final PrintStream output = System.out;

    public static void main(String[] args) {

        StringBuffer results = new StringBuffer();

        int numberOfSets = input.nextInt();

        for (int i = 0; i < numberOfSets; ++i) {
            int a = input.nextInt();
            int b = input.nextInt();

            if (a < b) {
                results.append("<\n");
            } else if (a > b) {
                results.append(">\n");
            } else {
                results.append("=\n");
            }
        }
        System.out.print(results.toString());
    }
}

File: Chapter_01_Introduction/Getting_Started_The_Easy_Problems/Super_Easy/11172-Relational_Operator/main.cpp
/*
 * Problem 11172 Relational Operator
 *
 * Take in 2 numbers and output the relation that is appropriate
 * for the given numbers.
 */
#include <iostream>
#include <sstream>

int main() {
    std::ios::sync_with_stdio(0);
    std::cin.tie(0);

    std::ostringstream output;

    int testCases;
    std::cin >> testCases;
    while (testCases--) {
        int a, b;
        std::cin >> a >> b;
        if (a == b) {
            output << "=\n";
        } else {
            output << (a < b ? '<' : '>') << '\n';
        }
    }
    std::cout << output.str();
}
